### PR TITLE
New banner for Community Day

### PR DESCRIPTION
**Pull request by AlbaRoza**
Temporary banner until the community day happens (22nd of March)

### CODE REVIEW

**Review by AlbaRoza — COMMENTED** [review]
Link changed

File: src/components/Payoff/index.en.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-        <StyledSubParagraph><a href="/en/#about-signalen">More information on Signalen</a></StyledSubParagraph>
+        <StyledHeading>Save the date: Signalen Community Meetup</StyledHeading>
+        <StyledParagraph>Online edition. March, 22. </StyledParagraph>
+        <StyledSubParagraph>This edition is specifically for municipalities that are considering using Signalen to receive and handle their nuisance reports. In addition, during the meetup we will explore with the market how to ensure a sustainable ecosystem around Signalen.</StyledSubParagraph>
+        <StyledSubParagraph><a href="/en/news/2021-02-26-save-the-date-community-meeting">More information on the meetup</a></StyledSubParagraph>
       </StyledTextContainer>
     </StyledIntro>
     <StyledImage>
-      <ResponsiveImage src={missedChances} alt="Illustration of man with headphone and woman with mobile phone walking down the street" />
+      <ResponsiveImage src={city} alt="City with construction workers, people walking by and a hadicapped man" />
```
Comment (felixfaassen) [inline src/components/Payoff/index.en.js:17]: @AlbaRoza great PR! There is one small problem. You are referencing a variable {city} which does not exist and will break the build. Consider changing either line 4 with `import missedChances` to `import city` OR chance in this line {city} to {missedChances}. If you need help let me know.

**Review by felixfaassen — CHANGES_REQUESTED** [review]
Refer to the correct variable name to refer to city.png

File: src/components/Payoff/index.en.js (hunk 1 above)
Comment (felixfaassen) [reply src/components/Payoff/index.en.js]: @AlbaRoza I made the changes for you.

**Review by felixfaassen — DISMISSED** [review]

**Comment by felixfaassen** [comment]
@AlbaRoza and I decided to go for the solution provided by @bartjkdp . Thanks for the work @AlbaRoza

**Pull request closed without merging**